package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-demo-calculation-service",
   "description": "Digital service mock to claim public money in the event property subsides into mine shaft.",
-  "version": "2.0.11",
+  "version": "2.1.00",
```

Comment:
I think there is typo here. `2.1.0` rather than `2.1.00`

Reply:
not sure what happened here - now changed